virttest/qemu_vm.py

```diff
@@ -3575,6 +3575,12 @@ def _get_pci_bus(self, params, dtype=None, pcie=False):
             return {'aobject': '%s-0' % bridge_type}
         return {'aobject': params.get('pci_bus', 'pci.0')}
 
+    def _get_css_bus(self, params):
```

Comment:
Seems we can remove it.

Reply:
updated, sorry to only delete on local invironment

Reply:
Modified